website/screens/components/alert/code/AlertCodePage.tsx

```diff
@@ -16,22 +16,26 @@ const sections = [
         <thead>
           <tr>
             <th>Name</th>
+            <th>Type</th>
+            <th>Description</th>
             <th>Default</th>
-            <HeaderDescriptionCell>Description</HeaderDescriptionCell>
           </tr>
         </thead>
         <tbody>
           <tr>
-            <td>type: 'info' | 'confirm' | 'warning' | 'error'</td>
+            <td>type</td>
             <td>
-              <Code>'info'</Code>
+              <TableCode>'info' | 'confirm' | 'warning' | 'error'</TableCode>
             </td>
             <td>Uses on of the available alert types.</td>
+            <td>
+              <TableCode>'info'</TableCode>
+            </td>
           </tr>
           <tr>
-            <td>mode: 'inline' | 'modal'</td>
+            <td>mode</td>
             <td>
-              <Code>'inline'</Code>
+              <TableCode>'inline' | 'modal'</TableCode>
             </td>
             <td>
               Uses on of the available alert modes:
```

Comment:
Same as above, should be 'Uses **one** of the available alert types' instead of 'Uses on of the available alert types'?

Reply:
Yes! I fix this typo, thanks 🚀